aiooncue/__init__.py

```diff
@@ -78,13 +78,17 @@
     [[NAME_TO_SENSOR_ID[name] for name in ALL_DETAILS_NAMES]], separators=(",", ":")
 )
 
+LOGIN_SESSION_EXPIRED = 1200
+LOGIN_INVALID_USERNAME = 1202
+LOGIN_INVALID_PASSWORD = 1207
+
 LOGIN_FAILED_CODES = {
     0: "Unknown",
-    1200: "Session Expired",
-    1202: "Invalid username",
-    1207: "Invalid Password",
+    LOGIN_SESSION_EXPIRED: "Session Expired",
+    LOGIN_INVALID_USERNAME: "Invalid username",
+    LOGIN_INVALID_PASSWORD: "Invalid Password",
 }
-INCORRECT_CREDENTIALS_CODES = {1207, 1202}
+INCORRECT_CREDENTIALS_CODES = { LOGIN_INVALID_PASSWORD, LOGIN_INVALID_USERNAME }
```

Comment:
I think the simple solution is to not treat `1202` aka `LOGIN_INVALID_USERNAME ` as a credentials problem. I don't think users can change their username anyways

Reply:
Like it, I've updated and simplified